index.js

```diff
@@ -173,4 +173,23 @@ for (let i = 0; i < projects.length; i += 1) {
       body.removeChild(popup);
     });
   });
-}
\ No newline at end of file
+}
+
+// Validate contact form
```

Comment:
- [ ] Your code look's and work perfectly, but it's a requirement that the error message should show up close to the "submit" button, please consider changing the display position of the error message

![Screenshot_20220512_193033](https://user-images.githubusercontent.com/25227022/168144220-3fb74100-5abd-45c4-8dd6-0b2646f1cc41.png)